src/NServiceBus.Core.Tests/Msmq/QueuePermissionsTests.cs

```diff
@@ -13,7 +13,7 @@ public class QueuePermissionsTests
     {
         StringBuilder logOutput;
         string testQueueName = "NServiceBus.Core.Tests.QueuePermissionsTests";
-
+    
         [OneTimeSetUp]
```

Comment:
@kbaley intended for this to be a OneTimeSetup for the logger.  However, the problem was that the in the the log string wasn't being cleared on teardown.